<launch>
	<node name="grsim" pkg="roboteam_utils" type="grsim.sh" />
</launch>
